### ISSUE TITLE
the converted model cannot be opened with 3ds max

### ISSUE THREAD

**Issue by Sruimeng** [jimmiebergmann/fbx-file#1]
How can I open it with 3ds max?

**Comment by Sruimeng** [comment]
[outFbx1.zip](https://github.com/jimmiebergmann/fbx-file/files/4857293/outFbx1.zip)
this is my output fbx model

**Comment by jimmiebergmann** [comment]
Do you know what is wrong with the file?
Is it the file structure? Maybe you're adding records with incorrect names/child records?
I'm getting 3 boxes when I open the file in Blender 2.79b.

**Comment by Sruimeng** [comment]
Yes,I can also open this model in blender2.82, but in 3ds max2016 and It's not opening in 3ds max 2020, is there any special settings needed?and how to adding records with incorrext namnes/child records?

**Comment by VoidScute** [comment]
The only things to check it s if the fbx file is fbx valid. Also some software not open binary or ascii fbx file. Other advice : stay away from proprietary software you always had trouble and instead of libre software you can NOT fix it yourself and several time editor never fix bugs.

**Comment by ivan-ushakov** [comment]
Yes, current version has problem with FBX SDK and FBX Review app for macOS. I fixed this problem in #3.
